javascript/tsconfig.json

```diff
@@ -15,5 +15,5 @@
     "outDir": "./dist"
   },
   "include": ["src/**/*", "test/**/*"],
-  "exclude": ["./dist/**/*", "./node_modules"]
+  "exclude": ["./dist/**/*", "./node_modules", "./src/**/*.deno.ts"]
```

Comment:
Rather than putting this in here and then having an `exclude` in all the `config/*` files could we have another `deno` specific config file like `config/deno.config.ts`?

Reply:
Oh, hmm, maybe I've misunderstood things, this is actually fine :grimacing: